.azure-pipelines/markers-check.yml

```diff
@@ -0,0 +1,12 @@
+steps:
+- script: |
+    set -x
+
+    pip3 install natsort
+
+    python3 ./.azure-pipelines/markers_check/markers_check.py tests
+    if [[ $? -ne 0 ]]; then
+      echo "##vso[task.complete result=Failed;]Markers check fails."
```

Comment:
Why set variable `result` here?

Reply:
As I searched, `##vso[task.complete result=Failed;]` is used to explicitly mark a task as failed. And exit 1 is a double confirm to fail the pipeline.